src/rabbitmanager.js

```diff
@@ -158,6 +158,17 @@ class RabbitManager {
     const uriEncodedComponents = this.encodeURIComponents({name: name, vhost: vhost});
     return await this.request(`queues/${uriEncodedComponents.vhost}/${uriEncodedComponents.name}`, {method: 'delete'});
   }
+
+  async messagesFromQueue(queueName, options={count: 5, requeue: true, encoding:'auto'}, vhost='/') {
```

Comment:
Perhaps add a http://usejsdoc.org/ comment to this function telling us what it does? It's not obvious from the name.